ssh/tailssh/tailssh.go

```diff
@@ -827,7 +844,7 @@ type sshConnInfo struct {
 	// pubKey is the public key presented by the client, or nil
 	// if they haven't yet sent one (as in the early "none" phase
 	// of authentication negotiation).
-	pubKey ssh.PublicKey
+	_pubKey ssh.PublicKey
```

Comment:
what's going on here with the underscore!?

Reply:
Oh this was supposed to be deleted, but I initially only renamed it to find usage breaks